tests/unit/CMakeLists.txt

```diff
@@ -9,11 +9,8 @@
 # terms of the BSD-3 license. We welcome feedback and contributions, see file
 # CONTRIBUTING.md for details.
 
-# Include the top mfem source directory - needed by some tests, e.g. to
-# #include "general/text.hpp".
-include_directories(BEFORE ${PROJECT_SOURCE_DIR})
-# Include the build directory where mfem.hpp and mfem-performance.hpp are.
-include_directories(BEFORE ${PROJECT_BINARY_DIR})
```

Comment:
Is this really not needed?

Reply:
I added a `target_include_directories` statement for the mfem library target, so the correct includes should be passed to any cmake targets that depend on mfem.